tests/test_jax_nonbonded.py

```diff
@@ -191,12 +184,12 @@ def generate_random_inputs(n_atoms, dim, instance_flags=difficult_instance_flags
     if instance_flags["randomize_beta"]:
         beta += rand()
 
-    args = (conf, params, box, lamb, charge_rescale_mask, lj_rescale_mask, beta, cutoff)
+    args = (conf, params, box, charge_rescale_mask, lj_rescale_mask, beta, cutoff)
 
     return args
 
 
-def compare_two_potentials(u_a: NonbondedFxn, u_b: NonbondedFxn, args: NonbondedArgs, differentiate_wrt=(0, 1, 3)):
+def compare_two_potentials(u_a: NonbondedFxn, u_b: NonbondedFxn, args: NonbondedArgs, differentiate_wrt=(0, 1, 2)):
```

Comment:
Good catch, we were not differentiating wrt box before. I'm not sure there's harm in doing this, but since we don't support dU/dbox in the custom ops potentials it probably doesn't make sense to test this here.

Reply:
Fixed in d67f3de